apps/ideas/src/components/modules/WorkshoppingPage/CommentForm.js

```diff
@@ -60,22 +93,28 @@ function CommentForm(props) {
             src={`https://picsum.photos/70?random=${props.id}`}
           />
           <textarea
+            className='input_color2'
             onKeyUp={(e) => {
               e.target.style.height = 'inherit';
-              e.target.style.height = `${e.target.scrollHeight}px`; 
+              e.target.style.height = `${e.target.scrollHeight}px`;
               // In case you have a limitation
               // e.target.style.height = `${Math.min(e.target.scrollHeight, limit)}px`;
               //textCounter(this,'counter',250);
             }}
-            style={{width:"100%", overflow:"hidden"}}
+            style={{ width: '100%', overflow: 'hidden', borderColor: borderColorComment}}
             name="text"
-            placeholder="What are your thoughts? (max 250 characters)"
+            placeholder="What are your thoughts? (min 25 characters - max 250 characters)"
             value={props.handleTextChange}
             onChange={handleTextChange}
-            maxlength={MAX_COMMENT_CHARS}></textarea>
+            maxlength={MAX_COMMENT_CHARS}
+            minlength={MIN_COMMENT_CHARS}
+            // required
+          ></textarea>
           {/* source: https://codepen.io/patrickwestwood/pen/gPPywv */}
           <div id="the-count">
             <span id="chars-left">{charsLeft}</span>
+            <br />
```

Comment:
Could we remove this `<br />` tag and add some styling to achieve the same result? (or we may be able to use `<p>` tags to get them on separate lines)

source: https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br